packages/ag-charts-enterprise/src/features/zoom/zoom.ts

```diff
@@ -269,7 +271,7 @@ export class Zoom extends _ModuleSupport.BaseModuleInstance implements _ModuleSu
             const { direction } = hoveredAxis;
             const axisZoom = direction === ChartAxisDirection.X ? x : y;
             this.updateAxisZoom(direction, axisZoom);
-        } else if (paddedRect?.containsPoint(event.offsetX, event.offsetY)) {
+        } else if (paddedRect?.containsPoint(event.offsetX, event.offsetY) && !event.preventZoomDblClick) {
```

Comment:
Note: This logic replicates a behaviour that was lost in https://github.com/ag-grid/ag-charts/commit/e9e97720c93c667308dce54dda80b615846d60ea